package.json

```diff
@@ -64,7 +63,8 @@
     "redux-thunk": "^2.1.0",
     "request": "^2.79.0",
     "winston": "^2.3.0",
-    "winston-daily-rotate-file": "^1.4.2"
+    "winston-daily-rotate-file": "^1.4.2",
+    "yargs": "^6.4.0"
```

Comment:
Added yargs as an explicit dependency to consume configuration settings from command-line arguments. It's already used by WebPack, but was not previously in package.json or in the production bundle.

We should be able to remove it from production dependencies when we start configuring deployed applications with environment variables only (I've dropped this objective for this pull request because there is a lot more work needed).